spec/routing/vendors_routing_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe VendorsController do
+  describe "routing" do
+    it "routes to #index" do
+      expect(get: "/vendors").to route_to("vendors#index")
+    end
+
+    it "routes to #show" do
+      expect(get: "/vendors/1").to route_to("vendors#show", id: "1")
+    end
+
+    it "does not route to #edit" do
+      expect(get: "/vendors/1/edit").not_to be_routable
+    end
+
+    it "routes to #create" do
+      expect(post: "/vendors").to route_to("vendors#create")
+    end
+
+    it "does not route to #update via PUT" do
+      expect(put: "/vendors/1").not_to be_routable
+    end
+
+    it "does not route to #update via PATCH" do
+      expect(patch: "/vendors/1").not_to be_routable
+    end
+
+    it "routes to #destroy" do
+      expect(delete: "/vendors/1").to route_to("vendors#destroy", id: "1")
+    end
+  end
+end
```

Comment:
I wonder if it's valuable using routing specs when we could/do use request specs already (which test both routing and controller behavior).

Reply:
I find routing specs helpful because I'm still not comfortable with the DLS for config/routes.rb.   

The vendor request specs are all skipped for now - I wrote no controller code -- what is there is from scaffold generator.